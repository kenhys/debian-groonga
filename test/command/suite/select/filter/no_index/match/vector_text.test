table_create Memos TABLE_NO_KEY
column_create Memos tags COLUMN_VECTOR Text

load --table Memos
[
{"tags": ["Groonga", "Rroonga", "Mroonga"]}
]

select Memos --filter 'tags @ "Rroonga"'
