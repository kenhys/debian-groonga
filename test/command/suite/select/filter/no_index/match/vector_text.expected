table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos tags COLUMN_VECTOR Text
[[0,0.0,0.0],true]
load --table Memos
[
{"tags": ["Groonga", "Rroonga", "Mroonga"]}
]
[[0,0.0,0.0],1]
select Memos --filter 'tags @ "Rroonga"'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "tags",
          "Text"
        ]
      ],
      [
        1,
        [
          "Groonga",
          "Rroonga",
          "Mroonga"
        ]
      ]
    ]
  ]
]
